src/search/forbid_iterative_search.cc

```diff
@@ -764,6 +779,64 @@ void ForbidIterativeSearch::add_forbid_plan_reformulation_option(OptionParser &p
 
 }
 
+//////////////// De-duplication stuff ///////////////////////////
+
+std::string ForbidIterativeSearch::get_original_name(std::string act_name) const {
+        string del = duplicate_prefix, orig_act_name;
+        int start, end = act_name.find(del);
```

Comment:
The variable start seems to be used only once and is not initialized. 
Maybe there is no need for the variable, just use 0 in the find function?